src/lambda/handler-runner/HandlerRunner.js

```diff
@@ -79,6 +82,11 @@ export default class HandlerRunner {
       )
     }
 
+    if (supportedInvoke.has(runtime)) {
+      const { default: InvokeRunner } = await import('./invoke-runner/index.js')
+      return new InvokeRunner(this.#funOptions, this.#serverless, this.#env)
```

Comment:
Why is it called an `InvokeRunner` and not `RustRunner`?

Reply:
I initially planned to name it RustRunner. But while implementing it turned out to be not very rust specific. It should work with lambci/lambda:provided in general. Though I haven't come acorss any except rust.

If necessary, I can change its name to RustRunner.